prime-router/src/testIntegration/kotlin/datatests/mappinginventory/xtn/XTNTests.kt

```diff
@@ -3,9 +3,11 @@ package gov.cdc.prime.router.datatests.mappinginventory.xtn
 import gov.cdc.prime.router.datatests.mappinginventory.translateAndCompareFHIRToHL7
 import gov.cdc.prime.router.datatests.mappinginventory.translateAndCompareHL7ToFHIR
 import org.junit.jupiter.api.Test
+import kotlin.test.Ignore
 
 class XTNTests {
     // Tests use (line 2)
+    @Ignore
```

Comment:
Why do we want to keep and ignore rather than delete?

Reply:
Yep will delete before merging